TRIAGE.md

```diff
@@ -0,0 +1,49 @@
+# Triage in OpenSearch Core
+## What Do We Do With These Labels Anyway?
```

Comment:
The tone is a little casual to my taste, consider changing this title

Reply:
Sure. Whatever, man. ;)  
```suggestion
## Why are there Specialized Labels in Core?
```

Reply:
We actually do have a style guide for the whole project in this regard: https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#tone